tests/tests_suite_test.go

```diff
@@ -94,7 +94,7 @@ var _ = SynchronizedBeforeSuite(testsuite.SynchronizedBeforeTestSetup, testsuite
 
 var _ = SynchronizedAfterSuite(testsuite.AfterTestSuitCleanup, testsuite.SynchronizedAfterTestSuiteCleanup)
 
-var _ = AfterEach(func() {
+var _ = AfterEach(OncePerOrdered, func() {
```

Comment:
in the commit message, I think it is important to warn that `OncePerOrdered` may be used only when no `It` section modifies the shared VM.

Reply:
@rhrazdil, since you are also working on something similar
You might want to check this out please

@qinqon as well please since you are familiar with it

Reply:
> in the commit message, I think it is important to warn that OncePerOrdered may be used only when no It section modifies the shared VM.

I'm not sure I see why no `It` may modify the shared VM, Ordered tests are executed in `Serial` fashion, on the same process. Snippet from ginkgo docs:
```
Specs in an Ordered container may run in parallel with respect to other specs, but they 
will always run sequentially on the same parallel process. This allows specs in Ordered 
containers to rely on mutating local closure state.
```